docker/api/container.py

```diff
@@ -804,7 +804,7 @@ def logs(self, container, stdout=True, stderr=True, stream=False,
                 datetime or integer epoch (in seconds)
 
         Returns:
-            (generator or str)
+            (generator iterator of output bytes or bytes)
```

Comment:
I think that you are mixing 2 concerns when using the term `generator iterator`. They are 2 different things. Just using `generator` should be fine.
My proposal would be:

`(generator of bytes or bytes)`